boards.txt

```diff
@@ -423,6 +423,62 @@ magicbit.menu.UploadSpeed.921600.upload.speed=921600
 magicbit.menu.UploadSpeed.115200=115200
 magicbit.menu.UploadSpeed.115200.upload.speed=115200
 ##############################################################
+
+S_ODI_Ultra.name=S.ODI Ultra v1
+
+S_ODI_Ultra.upload.tool=esptool_py
+S_ODI_Ultra.upload.maximum_size=1310720
+S_ODI_Ultra.upload.maximum_data_size=327680
+S_ODI_Ultra.upload.wait_for_upload_port=true
+
+S_ODI_Ultra.serial.disableDTR=true
+S_ODI_Ultra.serial.disableRTS=true
+
+S_ODI_Ultra.build.mcu=esp32
+S_ODI_Ultra.build.core=esp32
+S_ODI_Ultra.build.variant=S.ODI_Ultra_v1.0
```

Comment:
this value may not be accepted by GCC as a valid pre-processor definition. Replace the "." with "_".